lib/util.ml

```diff
@@ -1,5 +1,3 @@
-(* Repeat a non-determistic property                          *)
```

Comment:
To be completely exact, this repeat a property, and it is useful when the property is non-deterministic.
Not sure it is that important though :-)

Reply:
OK, point well-taken :+1:
This comment is removed by the PR, so it is in a sense already addressed :slightly_smiling_face: